04-advanced-topics/network-analysis_rnaseq_01_wgcna.Rmd

```diff
@@ -286,6 +291,23 @@ df <- round(df) %>%
   dplyr::filter(rowSums(.) >= 50)
 ```
 
+Another thing we need to do is make sure our main experimental group label is set up. 
+In this case `refinebio_treatment` has two groups: `pre-adt` and `post-adt`. 
+We need this to be a factor, but we also need `pre-adt` as the first level. 
```

Comment:
Clarifying the conversion to a factor (we don't _need_ it, but it is nice to do)
```suggestion
To keep these two treatments in logical (rather than alphabetical) order, we will convert this to a factor with `pre-adt` as the first level. 
```